inspect_model.ipynb

```diff
@@ -35,11 +35,11 @@
     "import matplotlib.pyplot as plt\n",
     "import matplotlib.patches as patches\n",
     "\n",
-    "import utils\n",
-    "import visualize\n",
-    "from visualize import display_images\n",
-    "import model as modellib\n",
-    "from model import log\n",
+    "from mask_rcnn import utils\n",
+    "from mask_rcnn import visualize\n",
+    "from mask_rcnn from visualize import display_images\n",
```

Comment:
from mask_rcnn from visualize construct seems odd?

Reply:
yes, but it was there before... :) OK, I will correct it